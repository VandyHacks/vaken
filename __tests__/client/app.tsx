import React from 'react';
import renderer from 'react-test-renderer';

import Vaken from '../../src/client/app';

describe('Test Vaken', () => {
	it('Vaken main app renders correctly', async () => {
		const component = renderer.create(<Vaken />).toJSON();
		expect(component).toMatchSnapshot();
	});
});
