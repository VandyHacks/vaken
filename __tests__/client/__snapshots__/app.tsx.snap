// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Vaken Vaken main app renders correctly 1`] = `null`;
